src/flashcart/ed64/ed64.c

```diff
@@ -15,45 +15,79 @@
 
 // #include "../menu/settings.h"
 
-// // This is a trial hack before using the settings API.
-// #ifndef LAST_ROM_FILE_PATH
-// #define LAST_ROM_FILE_PATH "/ed64/last_rom.txt"
-// #endif
-
+// This is a trial hack before using the settings API.
+#ifndef LAST_SAVE_FILE_PATH
+#define LAST_SAVE_FILE_PATH strip_sd_prefix("/menu/last_rom.txt")
```

Comment:
```suggestion
#define LAST_SAVE_FILE_PATH "/menu/last_rom.txt"
```

Reply:
Are you sure that the `strip_sd_prefix` is needed? it should not be... If it is, it should be used where required and not in the define.

Reply:
Extra note, the `strip_sd_prefix` is used to "strip" the file path of `sd:` for which we do not use here (or probably most other places in this PR).